src/SCSS/details.scss

```diff
@@ -0,0 +1,80 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap');
+
+.details-image {
+  width: 300px;
+  height: 280px;
+}
+
+.details-container {
+  margin-top: 30px;
+  margin-left: 30px;
+}
+
+.button-container {
+  display: flex;
+  height: 50px;
+  width: 10rem;
+  background-color: #98c00d;
+  border-radius: 25px 25px 25px 25px;
+  justify-content: center;
+  align-items: center;
+  margin-top: 5rem;
+}
+
+.reserve-button {
+  background: none;
+  color: white;
+  border: none;
+  padding: 0;
+  font: inherit;
+  cursor: pointer;
+  outline: inherit;
+}
+
+.color-container {
+  display: none;
+}
+
+@media only screen and (min-width: 678px) {
+  .details-container {
+    display: flex;
+    gap: 10rem;
+    margin-left: 3rem;
+  }
+
+  .info-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    font-family: 'Poppins', sans-serif;
+  }
```

Comment:
Please make sure to follow the [behance design](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign):
As seen below, the details are displayed in a container with the text aligned on one side and the value on the end. 
| Your design  | Behance design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/198154554-a9cf4e07-802a-4692-89a3-62796fc62c84.png)  | ![image](https://user-images.githubusercontent.com/42799579/198154600-b0fd15da-f858-4c06-b3f6-2b95ecfffaef.png)  |

Also, take into account the two different background colors.

The same applies to the color picker:
| Your design  | Behance design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/198155694-26745ebe-d0d3-4476-8dc2-a1610039d824.png)  | ![image](https://user-images.githubusercontent.com/42799579/198155746-051ccfe6-db86-4f1a-a833-a9bea8597eea.png)  |